blockchain/chain/src/store/chain_store.rs

```diff
@@ -201,6 +201,9 @@ where
 
     /// Returns Tipset from key-value store from provided cids
     pub async fn tipset_from_keys(&self, tsk: &TipsetKeys) -> Result<Arc<Tipset>, Error> {
+        if tsk.cids().is_empty() {
+            return Ok(self.heaviest_tipset().await.unwrap());
```

Comment:
Would be nice if this was handled, although I get this should never happen (we should maybe looking into changing this from being an Optional value and always initializing with a heaviest, initializing with genesis ts so we don't have to handle these